src/schema.rs

```diff
@@ -157,6 +163,17 @@ pub enum Type {
     String(SizeLength),
     ContractName(SizeLength),
     ReceiveName(SizeLength),
+    /// An unsigned integer encoded using LEB128 with the addition of a
```

Comment:
Consider adding doc comments to all the variants of `Type` now that you've started ;)